src/comp/ScrollableTagList.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+
+import style from "css/comp/ScrollableTagList.module.css";
+import ProjectTag from "./ProjectTag";
+
+type ProjectTagListProps = {
+  tagIDs: string[];
+  onTagClick: (
+    tagID: string,
+    e: React.MouseEvent<HTMLParagraphElement>
+  ) => void;
```

Comment:
```suggestion
  onTagClick?: (
    tagID: string,
    e: React.MouseEvent<HTMLParagraphElement>
  ) => void;
```
指定しない場合を許容していないのでは(既視感)